documentation/cgroups_in_storm.md

```diff
@@ -0,0 +1,61 @@
+               }
+               admin {
+                      uid = 500;
+                      gid = 500;
+               }
+       }
+       cpu {
+       }
+}
+```
+
+# Settings Related To CGroups in Storm
+
+| Setting                       | Function                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cgroup.enable                 | This config is used to set whether or not cgroups will be used.  Set "true" to enable use of cgroups.  Set "false" to not use cgroups. When this config is set to false, unit tests related to cgroups will skipped. Default set to "false"                                                                                                                                                                                                                                                                                         |
+| cgroup.storm.hierarchy.dir    | The path to the cgroup hierarchy that storm will use.  Default set to "/cgroup/storm_resources"                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|     cgroup.storm.resources    | A list of subsystems that will be regulated by CGroups. Default set to cpu and memory                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
What other subsystems are valid?  If there are no other ones that are valid, simply indicate that.